frontend/glideinFrontendElement.py

```diff
@@ -877,15 +875,14 @@ def iterate_one(self):
             callback_creds = self.credentials_plugin.get_credentials(credential_purpose=CredentialPurpose.CALLBACK)
             if not callback_creds:
                 logSupport.log.debug("Custom callback credential not provided. Using default.")
-                tkn_dir = TOKEN_DIR
-                if not os.path.exists(tkn_dir):
-                    os.mkdir(tkn_dir, 0o700)
+                if not os.path.exists(TOKEN_DIR):
+                    os.mkdir(TOKEN_DIR, 0o700)
                 callback_generator = create_credential(
                     "IdTokenGenerator",
                     cred_type=CredentialType.DYNAMIC,
                     purpose=CredentialPurpose.CALLBACK,
                     trust_domain=trust_domain,
-                    context={"cache_dir": tkn_dir},
+                    context={"cache_dir": TOKEN_DIR},
```

Comment:
Is `TOKEN_DIR` the right one to use for setting context for `"cache_dir"` attribute? The names `cache` and `token` makes me think otherwise.